.github/workflows/release-build.yml

```diff
@@ -0,0 +1,80 @@
+# Releases a new minor / major version from the HEAD of the main branch
+name: Release Build
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.0, 1.2.1-alpha.1
+        required: true
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 11
+      - uses: burrunan/gradle-cache-action@v1
+        name: Build
+        with:
+          arguments: --stacktrace build
+          remote-build-cache-proxy-enabled: false
+      - uses: actions/upload-artifact@v2
+        name: Save unit test results
+        if: always()
+        with:
+          name: test-results
+          path: jmx-metrics/build/reports/tests/test
+  integration-test:
+    name: integration-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 11
+      - uses: burrunan/gradle-cache-action@v1
+        name: Integration Tests
+        with:
+          arguments: --stacktrace integrationTest
+          remote-build-cache-proxy-enabled: false
+      - uses: actions/upload-artifact@v2
+        name: Save integrationTest results
+        if: always()
+        with:
+          name: integration-test-results
+          path: jmx-metrics/build/reports/tests/test
```

Comment:
Just checking: is the report path for `integrationTest` the same as for `test`?

Reply:
Right we currently override the default `test` task. I want to change to test-sets eventually